src/ListContainer.cpp

```diff
@@ -40,20 +42,18 @@ extern thread_local std::string thread_id;
 
 // IMPLEMENTATION
 
+
 // Constructor - set default values
-ListContainer::ListContainer()
-{
+ListContainer::ListContainer() {
```

Comment:
Codacy found an issue: [Member variable 'ListContainer::filedate' is not initialized in the constructor.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238852)